modules/decklinksrc/decklinksrc.py

```diff
@@ -1,40 +1,42 @@
 import cv2 as cv2
 import numpy as np
+import threading
+
 
 class DeckLinkSRC:
 
-    def __init__(self):
+    def __init__(self, videoPipeline):
         self.__currentFrame = None
-        self.capture = cv2.VideoCapture('decklinkvideosrc mode=7 connection=0 ! videoconvert ! appsink') #Starts capture on initialization of object
-        #Because of this, we no longer need start stream code from Aryan
+        self.capture = cv2.VideoCapture(
+            'decklinkvideosrc mode=7 connection=0 ! videoconvert ! appsink')  # Starts capture on initialization of object
+        self.videoPipeline = videoPipeline
+        main = threading.Thread(target=self._main_)
```

Comment:
Improve naming (e.g. `mainThreadDeckLinkSRC`).

Reply:
Got it